### PR TITLE
feat: Add Ses lambda event support to Parser utility #213

### PR DESCRIPTION
**Pull request by risenberg-cyberark**
Also added email-validator module as an extra dependancy for pydantic email addresses validations.

### CODE REVIEW

File: aws_lambda_powertools/utilities/parser/models/ses.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+from pydantic.networks import EmailStr
+from pydantic.types import PositiveInt
+from typing_extensions import Literal
+
+
+class SesReciptVerdict(BaseModel):
```
Comment (heitorlessa) [inline aws_lambda_powertools/utilities/parser/models/ses.py:10]: Typo - Receipt

File: pyproject.toml [hunk 2]
```diff
@@ -51,7 +51,7 @@ flake8-bugbear = "^20.1.4"
 
 
 [tool.poetry.extras]
-pydantic = ["pydantic", "typing_extensions"]
+pydantic = ["pydantic", "typing_extensions", "pydantic[email]"]
```
Comment (heitorlessa) [inline pyproject.toml:54]: What does Pydantic[email] brings? 

Anything else other than EmailStr?

**Review by heitorlessa — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (risenberg-cyberark) [reply pyproject.toml:54]: just that

File: aws_lambda_powertools/utilities/parser/models/ses.py (hunk 1 above)
Comment (risenberg-cyberark) [reply aws_lambda_powertools/utilities/parser/models/ses.py:10]: fixed

File: pyproject.toml [hunk 3]
```diff
@@ -24,6 +24,7 @@ fastjsonschema = "^2.14.5"
 boto3 = "^1.12"
 jmespath = "^0.10.0"
 pydantic = {version = "^1.6.0", optional = true }
+email-validator = "*"
```
Comment (heitorlessa) [inline pyproject.toml:27]: Could you make this optional, please?

**Review by heitorlessa — CHANGES_REQUESTED** [review]
One last minor change and it's good to merge :-)

File: pyproject.toml (hunk 3 above)
Comment (risenberg-cyberark) [reply pyproject.toml]: yes, fixed

**Review by heitorlessa — APPROVED** [review]
LGTM - Thanks a lot Ran!!

**Pull request merged**